pandas/core/common.py

```diff
@@ -515,7 +516,7 @@ def f(x):
 
 
 def convert_to_list_like(
-    values: Scalar | Iterable | AnyArrayLike,
+    values: Scalar | IndexLabel | AnyArrayLike,
```

Comment:
Why is this changed? In code we check for an `Iterable`. `IndexLabel` only includes `Sequence`

Reply:
Reverted it. Probably was causing an error with one of the other changes from #43744.